README.md

```diff
@@ -92,6 +93,65 @@ your app.config):
 The available configuration options for each backend are listed in their
 module's documentation.
 
+Multiple sinks
+--------------
+Previously, lager supported a single sink `lager_event`. Lager can now support
+additional sinks with their own backend configurations. Eventually, the goal is
+to support different latency and sync/async message policies. (For example, if
+there were an `audit` sink, we might be willing to pay the latency costs to log
+all of those messages to disk and we don't want lager to drop them in an
+overload scenario.)
+
+To use multiple sinks (beyond the built in sink of lager and lager_event), you
+need to:
+
+1. Setup rebar.config
+2. Configure the backends in app.config
+
```

Comment:
Yeah we definitely do. It's confusing and deserves clarity if not for external developers' sake - but our own!